main.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"context"
+	"strconv"
+	"time"
+
+	proxyruntime "github.com/OpenFunction/dapr-proxy/pkg/runtime"
+	"github.com/OpenFunction/dapr-proxy/pkg/utils"
+	ofctx "github.com/OpenFunction/functions-framework-go/context"
+	"github.com/OpenFunction/functions-framework-go/framework"
+	diag "github.com/dapr/dapr/pkg/diagnostics"
+	"github.com/dapr/dapr/pkg/modes"
+	"github.com/dapr/dapr/pkg/runtime"
+	"github.com/pkg/errors"
+	"k8s.io/klog/v2"
+)
+
+const (
+	defaultAppProtocol = "grpc"
+	protocolEnvVar     = "APP_PROTOCOL"
+	debugEnvVar        = "DEBUG"
+)
+
+var (
+	FuncRuntime *proxyruntime.Runtime
+)
+
+func main() {
+	ctx := context.Background()
+	fwk, err := framework.NewFramework()
+	if err != nil {
+		klog.Exit(err)
+	}
+
+	funcContext := utils.GetFuncContext(fwk)
+
+	host := utils.GetFuncHost(funcContext)
+	port, _ := strconv.Atoi(funcContext.GetPort())
+	protocol := utils.GetEnvVar(protocolEnvVar, defaultAppProtocol)
+	config := &proxyruntime.Config{
+		Protocol: runtime.Protocol(protocol),
+		Host:     host,
+		Port:     port,
+		Mode:     modes.KubernetesMode,
+	}
+
+	FuncRuntime = proxyruntime.NewFuncRuntime(config, funcContext)
+	if err := FuncRuntime.CreateFuncChannel(); err != nil {
+		klog.Exit(err)
+	}
+
+	if err := fwk.Register(ctx, EventHandler); err != nil {
+		klog.Exit(err)
+	}
+
+	if err := fwk.Start(ctx); err != nil {
+		klog.Exit(err)
+	}
+}
+
+func EventHandler(ctx ofctx.Context, in []byte) (ofctx.Out, error) {
+	debugVal := utils.GetEnvVar(debugEnvVar, "false")
+	debug, _ := strconv.ParseBool(debugVal)
+	start := time.Now()
+	defer func() {
+		if debug {
+			elapsed := diag.ElapsedSince(start)
+			klog.Infof("Input: %s - Event Forwarding Elapsed: %vms", ctx.GetInputName(), elapsed)
```

Comment:
klog use v to controller log output : 
https://pkg.go.dev/k8s.io/klog#V
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md

`klog.V(4).Info("Debug log")`

It's controlled by the `-v` flag 
-v=0
	Enable V-leveled logging at the specified level.